docs/zap-report.html

```diff
@@ -598,6 +598,20 @@ <h3>Alert Detail</h3>
   
   
   
+<tr bgcolor="#e8e8e8">
```

Comment:
The beta header is missing a word. You folks have already fixed this in `dev` on your end, but please include it in this pull request.

Reply:
Updated!